app/goal_routes.py

```diff
@@ -0,0 +1,87 @@
+from flask import Blueprint, jsonify, request, abort, make_response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from app.task_routes import validate_model
+from datetime import date
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+@goal_bp.route('', methods=['POST'])
+def create_one_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Nice work with try-except to handle invalid data, but it would be better to let the end-user know what data was invalid.  It's also a good opportunity to make it a helper function (shared with patch methods).